docs/user/rstudio/ide/guide/tools/version-control.qmd

```diff
@@ -14,9 +14,9 @@ RStudio supports the following open source version control systems:
 
 -   [Subversion](http://subversion.apache.org/)
 
-To use version control with RStudio, you should first ensure that you have installed Git and/or Subversion tools on your workstation (details below).
+To use version control with RStudio, first ensure that Git and/or Subversion tools are installed on your workstation (details below).
 
-Version control is most useful when used with a remote repository. Remote repositories are typically managed by your company or are hosted in the cloud (e.g. [Github](https://github.com/)). Make sure you have credentials to access these systems. If you only want to learn how to use version control, you can manage a standalone system on your workstation but you will not be able to share code with others.
+Version control is most useful when used with a remote repository. Remote repositories are typically managed by your company or are hosted in the cloud (e.g., [Github](https://github.com/) or [GitLab](https://about.gitlab.com/)). Make sure you have credentials to access these systems. If you only want to learn how to use version control, you can manage a standalone system on your workstation but you will not be able to share code with others.
 
 You should also become familiar with using [RStudio Projects](../authoring/projects.html) (which are required for version control features to be enabled).
 
```

Comment:
In the linux commands for installing git below, why `install git-core` rather than `install git`? I'm referencing this: https://git-scm.com/download/linux

Reply:
Great call! This was a quite old article - git-core used to be required as there was a namespace conflict with git. I've completely refactored this to the below, as those resources have detailed instructions for every platform.

> Prior to using Git with RStudio you should install it using the appropriate method for your platform as indicated at <https://git-scm.com/download/linux>.

and for subversion:

> Prior to using Subversion with RStudio you should install it using the appropriate method for your platform as indicated at <https://subversion.apache.org/packages.html>.